modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java

```diff
@@ -670,58 +679,62 @@ public long updateAndReturnSlotDeleteSafeZone(String nodeID, long safeZoneOfNode
      */
     public void clearAllActiveSlotRelationsToQueue(String queueName) {
 
-        if(log.isDebugEnabled()) {
-            log.debug("Clearing all slots for queue " + queueName);
-        }
+        if (HazelcastAgent.getInstance().isActive()) {
+            if(log.isDebugEnabled()) {
+                log.debug("Clearing all slots for queue " + queueName);
+            }
 
-        if (null != unAssignedSlotMap) {
-            unAssignedSlotMap.remove(queueName);
-        }
+            if (null != unAssignedSlotMap) {
+                unAssignedSlotMap.remove(queueName);
+            }
 
-        if (null != slotIDMap) {
-            slotIDMap.remove(queueName);
-        }
+            if (null != slotIDMap) {
+                slotIDMap.remove(queueName);
+            }
 
-        // Clear slots assigned to the queue along with overlapped slots
-        if (AndesContext.getInstance().isClusteringEnabled()) {
-            String nodeId = HazelcastAgent.getInstance().getNodeId();
+            // Clear slots assigned to the queue along with overlapped slots
+            if (AndesContext.getInstance().isClusteringEnabled()) {
```

Comment:
You don't need to check is clustering enabled here because this class will be initiated only in cluster mode. Therefore all the clustering enabled checking in this class are not necessary and they should be removed.